### PR TITLE
Issue #2578 Remove minikube ISO and related test

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

**Comment by agajdosi** [comment]
@praveenkumar Some minikube related code needs to be also removed from integration tests. Could I ask you for removing it, if it is ok for you? (Otherwise I will gladly take it in follow up issue... :)

Problematic parts are these:
https://github.com/minishift/minishift/blob/master/test/integration/testsuite/prepare.go#L74-L76
https://github.com/minishift/minishift/blob/master/test/integration/testsuite/prepare.go#L92-L94
https://github.com/minishift/minishift/blob/master/test/integration/testsuite/prepare.go#L99
https://github.com/minishift/minishift/blob/master/test/integration/testsuite/testsuite.go#L739-L740

**Comment by praveenkumar** [comment]
> Some minikube related code needs to be also removed from integration tests. Could I ask you for removing it, if it is ok for you? 

@agajdosi  Done

File: test/integration/testsuite/prepare.go [hunk 1]
```diff
@@ -89,15 +86,12 @@ func determineIsoFromFile(isoUrl string) string {
...
-	} else if matched, _ := regexp.MatchString(".*minikube\\.iso", isoUrl); matched {
-		fmt.Println("Minikube variant was assumed from the filename of ISO.")
-		isoName = "minikube"
 	} else if matched, _ := regexp.MatchString(".*centos7\\.iso", isoUrl); matched {
 		fmt.Println("CentOS variant was assumed from the filename of ISO.")
 		isoName = "centos"
 	} else {
-		fmt.Println("Can't assume ISO variant from its filename. Will use Boot2Docker. To avoid this situation please name your ISO to end with 'b2d.iso', 'centos7.iso' or 'minikube.iso'.")
-		isoName = "b2d"
+		fmt.Println("Can't assume ISO variant from its filename. Will use CentOS. To avoid this situation please name your ISO to end with 'b2d.iso' or 'centos7.iso'.")
```
Comment (agajdosi) [inline test/integration/testsuite/prepare.go]: @praveenkumar cool, thanks for making CentOS default :1st_place_medal:

**Comment by praveenkumar** [comment]
retest this please.

**Pull request merged**